addons/website/views/snippets/s_popup.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template id="s_popup" name="Popup">
-    <div class="s_popup o_snippet_invisible" data-vcss="001">
+    <div class="s_popup o_snippet_invisible" data-vcss="001" data-combine-drop-zones="true" data-show-modal-on-preview="true">
```

Comment:
We have many snippet with dataset whose info is only useful for the edit mode but they normally all are info about the snippet... here it's extra data users have to load for a pure editor behavior which does not depend on the snippet itself. Like if we added all data-drop-in on the snippet themself... :grimacing: 

There is better to do, I'll find a way :grimacing: